aries_cloudagent/resolver/default/indy.py

```diff
@@ -39,9 +39,10 @@ def _routing_keys_as_did_key_urls(routing_keys: Sequence[str]) -> Sequence[str]:
     did_key_urls = []
     for routing_key in routing_keys:
         if not routing_key.startswith("did:key:"):
-            did_key_urls.append(
-                DIDKey.from_public_key_b58(routing_key, KeyType.ED25519).key_id
-            )
+            did_key_urls.append(DIDKey.from_public_key_b58(routing_key, ED25519).key_id)
+            # NOTE: using hardcoded prefix ED25519 because py_multicodec library
+            # is outdated. Update after this PR gets released:
+            # https://github.com/multiformats/py-multicodec/pull/14
```

Comment:
I don't think this note is strictly relevant in this context since it applies more to the `ED25519` key type spec itself. Using `ED25519` in this way should not be impacted by the change noted :slightly_smiling_face: